code/modules/mob/dead/new_player/new_player.dm

```diff
@@ -510,7 +510,7 @@
 		qdel(src)
 
 /mob/dead/new_player/proc/ViewManifest()
-	var/dat = "<html><body>"
+	var/dat = "<html><head><meta http-equiv=\"Content-Type\" content=\"text/html; charset=UTF-8\"></head><body>"

```

Comment:
Why use " here ?

Reply:
I was originally doing that with all of them but then had a "wait a second" moment. Guess my regex missed this one.